README.adoc

```diff
@@ -13,109 +15,206 @@ Use this action to inform CloudBees Platform that an artifact version was create
 | Required?
 | Description
 
-| `digest`
-| String
-| No
-| The artifact digest that uniquely identifies the artifact.
-
-| `labels`
-| String
-| No
-| A comma-separated list of artifact labels.
-
 | `name`
 | String
 | Yes
-| The name of the artifact to send to the {PRODUCT} for artifact traceability purposes.
+| The name of the artifact reported to the CloudBees platform.
 
 | `url`
 | String
 | Yes
-| The URL where the artifact version is located.  For example, docker.io/myapp/myimg:1.0.0.
+| The URL where the artifact version is located.
+For example, `docker.io/myapp/myimg:1.0.0`.
 
 | `version`
 | String
 | Yes
-| The version of the artifact to send to the {PRODUCT} for artifact traceability purposes.
-
-| `repository-url`
-| String
-| No
-| The clone URL of the repository.
+| The version of the artifact reported to the CloudBees platform.
 
 | `commit`
 | String
-| No
+| Only required if a different repository/branch from the workflow is checked out.
```

Comment:
Some more explanation is required for these 4 repository/branch related fields since this action is not directly dealing with checking out code so the user might get confused here.
---
By default, CloudBees platform will associate the artifact version with the code commit associated with the workflow run in the workflow's repository/branch. But if a different commit/repository/branch was checked out for building the artifact, specify the commit ID that was used.